qutebrowser/keyinput/keyutils.py

```diff
@@ -298,22 +314,25 @@ class KeyInfo:
     Attributes:
         key: A Qt::Key member.
         modifiers: A Qt::KeyboardModifiers enum value.
+        text: A QString with the value of QKeyEvent::text()
```

Comment:
There's no `KeyInfo.text` attribute, right?

Reply:
Yes, thanks! This was left over from the previous approach.